lightkurve/injection.py

```diff
@@ -0,0 +1,288 @@
+"""Defines UniformDistribution, GaussianDistribution, TransitModel, and SupernovaModel"""
+
+import numpy as np
+from lightkurve import LightCurve
+import matplotlib.pyplot as plt
+from scipy.stats import norm
+
+class UniformDistribution(object):
+    """
+    Implements a class for choosing a value from a uniform distribution.
+
+    Attributes
+    ----------
+    lb : float
+        Lower bound of distribution
+    ub : float
+        Upper bound of distribution
+    """
+    def __init__(self, lb, ub):
+        self.lb = lb
+        self.ub = ub
+
+    def __repr__(self):
+        return 'UniformDistribution(lb={},ub={})'.format(self.lb,
+                                         self.ub)
+
+    def sample(self):
+        """Chooses values from uniform distribution.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        values : array-like
+            Returns array of randomly chosen values.
+        """
+        return np.random.uniform(self.lb, self.ub)
+
+    def plot(self):
+        """Plots specified distribution."""
+        t = np.arange(self.lb, self.ub, 0.01)
+        vals = [1]*len(t)
```

Comment:
There should be a space before and after the `*` here.

You can add a linter to your editor to to highlight these problems for you.